system-test/busybench/src/busybench.ts

```diff
@@ -15,7 +15,9 @@
  */
 
 import {writeFile} from 'fs';
+// eslint-disable-next-line node/no-extraneous-import
```

Comment:
Same questions as for other imports

Reply:
Yes; My best guess is that this happens because "pify" is a function rather than a package containing functions. But this was required to make the lint check pass.